src/update-ts-references.js

```diff
@@ -1,5 +1,5 @@
 const glob = require('glob');
-const path = require('path');
+const path = require('path').posix;
```

Comment:
I doubt this works because `process.cwd()` is still returing a windows style path.

Reply:
To be honest I thought the same 😄 and that is the reason why I first added a test run on a windows machine. 

The tests was before the change red because of your described behaviour with `\\` (for details see screenshot)

![image](https://user-images.githubusercontent.com/1651782/102936059-af86b400-44a7-11eb-8ea1-bd3e6274198b.png)

But now the tests a green and it seems that node.js can handle the combination.

Reply:
Even that would actually work I think is better to not rely on this internal behaviour (it was so tempting 😄 ). I now added some code by myself to ensure posix style.

Reply:
I tried it and it works. 👍  Thanks a lot!